.github/workflows/maintenance.yaml

```diff
@@ -0,0 +1,28 @@
+on:
```

Comment:
do we need this whole file?

Reply:
We've generally tried to avoid deviating from upstream by picking and choosing what we merge, but this might be one we want to nuke, if the actions running bothers us.

Reply:
right, it's the runs daily aspect i'm thinking of but it may not do anything and just generate noise which we just ignore.